src/app/components/users-groups/group-users/group-users.component.ts

```diff
@@ -0,0 +1,72 @@
+import {Component, OnInit} from '@angular/core';
+import {GroupServiceService} from '../../../services/group-service/group-service.service';
+import {RestApiServiceService} from '../../../services/restapiservice/rest-api-service.service';
+import {User} from '../../../models/User';
+import {Router} from '@angular/router';
+import {Group} from '../../../models/Group';
+import {DeleteDialogComponent} from '../../dialogs/delete-dialog/delete-dialog.component';
+import {MatDialog} from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-group-users',
+  templateUrl: './group-users.component.html',
+  styleUrls: ['./group-users.component.css']
+})
+export class GroupUsersComponent implements OnInit {
+
+  constructor(private router: Router,
+              private restApiService: RestApiServiceService,
+              private groupService: GroupServiceService,
+              public dialog: MatDialog) {
+  }
+
+  group: Group;
+  users: User[];
+  groupUsers: User[] = [];
+
+  ngOnInit(): void {
+    this.getGroup();
+    this.getAllUsers();
+  }
+
+  getGroup() {
+    let group = localStorage.getItem('groupName');
```

Comment:
mai bine salvezi tot obiectul in storage decat sa tot faci un request nou pe server sa iei valorile